src/magellan_firmware/firmware/magellan_controller/EncoderPublisher.h

```diff
@@ -1,6 +1,12 @@
 #ifndef ENCODER_PUBLISHER_H
 #define ENCODER_PUBLISHER_H
 
+// constants for quik maffs
+#define WHEEL_DIAMETER_METERS 0.10795       // meter
```

Comment:
@4ndr3w @noah710 doesnt this stuff belong in `config.h`?

Reply:
Good point, looks like there's already a value there: https://github.com/Pitt-RAS/racecar/blob/master/src/magellan_firmware/firmware/magellan_controller/config.h#L52

(You can also make those just `constexpr`)